homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+        <title>Home_Work1</title>
+    </head>
+    <body>
+        <article class="about_me_article">
+            <section class="about_me_section">
+                <h1 class="about_me_h1"> About me</h1>
+                <p class="about_me_text"> Hi! My name is Roman Lytvynenko and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.
+                </p>
+                <p
+                    class="about_me_text">This page was developed during the course <a class="about_me_links" href="https://www.mastersacademy.education/frontend-for-beginners-it"> 'Frontend for beginners' from Masters Academy in 2023. </a>
+                </p>
+                <p
+                    class="about_me_text"> This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio.
+                </p>
+                <p class="about_me_text"> You can contact me via
+                    <a class="about_me_links" href="https://google.com">Google</a> and/or check out my GitHub.
```

Comment:
pls add github link, and instead Google your social media pls

Reply:
done

Reply:
Should I close this pull request and create new one?

Reply:
Good job. Approved.